doc/users/index.rst

```diff
@@ -2,40 +2,111 @@
 
 .. redirect-from:: /contents
 
+..
+.. grid:: 3
 
-###########
-Users guide
-###########
+   .. grid-item-card:: Information
```

Comment:
Can you leave the card out and replace it with a regular title? I feel the double nesting of cards and dropdowns is a bit much.